Assets/MRTK/SDK/Features/UX/Scripts/BoundingBox/BoundingBox.cs

```diff
@@ -1148,7 +1148,18 @@ public IReadOnlyList<Transform> ScaleCorners
             get { return corners; }
         }
 
+        /// <summary>
+        /// Returns list of transforms pointing to the scale handle visuals of the bounding box.
+        /// </summary>
+        public IReadOnlyList<Transform> ScaleCornerVisuals
```

Comment:
i might be missing some important detail here: I can't spot you're using this anywhere in your fix. why are you adding those visuals and also exposing them to public API? If this is because of the tests you shouldn't add things to the public API of a sdk component just for test reasons - you can access these gameobjects as well by their name for testing purposes :)

Reply:
I didn't really want to access by name (searching for corners_i and then searching for the visuals game object under that object's children seems less ideal and more prone to breakage)